grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientOptions.java

```diff
@@ -173,8 +172,7 @@ public final class GrpcClientOptions {
      * to a gRPC {@link Status}.
      */
     public static final ClientOption<GrpcExceptionHandlerFunction> EXCEPTION_HANDLER =
-            ClientOption.define("EXCEPTION_HANDLER",
-                                (ctx, cause, metadata) -> GrpcStatus.fromThrowable(cause));
+            ClientOption.define("EXCEPTION_HANDLER", GrpcExceptionHandlerFunction.ofDefault());
```

Comment:
We don't use `UnwrappingGrpcExceptionHandleFunction` because this value is never used. Is that correct?

Reply:
`UnwrappingGrpcExceptionHandleFunction` should be the outermost handler so we don't use it here.

Reply:
Then, probably need to add a comment for that?
`GrpcExceptionHandlerFunction.ofDefault() isn't used because the actual exceptionHandler is set while building a gRPC client in GrpcClientBuilder.`

Reply:
Added comment, @minwoox can you check it again? Thanks,

Reply:
Based on [this change](https://github.com/line/armeria/pull/5571#discussion_r1605930750), we have to use it now.
Sorry but could you revert it? 😓

Reply:
Okay, I'll revert it